packages/frontend/core/src/pages/auth.tsx

```diff
@@ -198,25 +200,3 @@ export const loader: LoaderFunction = async args => {
   }
   return null;
```

Comment:
lack of redirect logic for `jumpToExpired`

Reply:
the logic is only suit for email verify routes, and it's already done